server/game/cards/characters/04/qhorinhalfhand.js

```diff
@@ -0,0 +1,37 @@
+const DrawCard = require('../../../drawcard.js');
+
+class QhorinHalfhand extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                afterChallenge: (event, challenge) => (
+                    challenge.challengeType === 'military' &&
+                    challenge.winner === this.controller &&
+                    challenge.isParticipating(this)
+                )
+            },
+            handler: () => {
+                this.game.promptForSelect(this.controller, {
+                    activePromptTitle: 'Select a character',
```

Comment:
Maybe the prompt title should be 'Select a character to kill' instead? It can be figured out from context since you were just prompted to activate Qhorin, but it still can be confusing if you're not clear why you're selecting a character.